src/app.css

```
@@ -2,96 +2,93 @@
 @tailwind components;
 @tailwind utilities;
 
-.force-white-text {
-	color: white !important;
-}
+@layer utilities {
+	.force-white-text {
+		color: rgb(var(--color-white)) !important;
+	}
 
-body {
-    transition: background-color 0.4s ease-in-out, color 0.4s ease-in-out;
+	.bg-gradient-primary {
+		background-image: linear-gradient(to right, var(--gradient-primary));
+	}
 }
 
-html {
-	color-scheme: dark;
-	--color-white: 255 255 255;
-	--color-black: 0 0 0;
-	--color-gray-50: 249 250 251;
-	--color-gray-100: 243 244 246;
-	--color-gray-200: 229 231 235;
-	--color-gray-300: 209 213 219;
-	--color-gray-400: 156 163 175;
-	--color-gray-500: 107 114 128;
-	--color-gray-600: 75 85 99;
-	--color-gray-700: 55 65 81;
-	--color-gray-800: 31 41 55;
-	--color-gray-900: 17 24 39;
-	--color-gray-950: 3 7 18;
+@layer base {
+	body {
+		transition: background-color 0.4s ease-in-out, color 0.4s ease-in-out;
+		background-image: linear-gradient(to bottom right, var(--background-gradient));
```

This looks like it fixes: https://github.com/codicocodes/dotfyle/pull/74#issuecomment-1793780927

```suggestion
		background-image: linear-gradient(to right, var(--background-gradient));
```